core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -43,7 +43,7 @@ <h1 class="stat-value"><[dashboardStats.average_ratings || 'N/A']></h1>
           <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
         </div>
         <div class="total-open-feedback stats-card">
-          <p class="stat-description">Open feedback</p>
+          <p class="stat-description">New feedback</p>
```

Comment:
I would lean towards calling it "Open feedback", actually. It seems like a more accurate name.